slides/06_Templates_Grundlagen.html

```diff
@@ -257,10 +262,10 @@ <h3>Beispiele</h3>
                          <button (click)="createPerson()">Neue Person</button>
                      </code>
                     <code class="hljs html" data-trim contenteditable>
-                         &ltaddress (customEvent)="machWas()"></address>
+                         <postal-address (customEvent)="machWas()"></postal-address>
                      </code>
                     <code class="hljs html" data-trim contenteditable>
-                         &ltpeople (deletePerson)="delete($event)"></people>
+                         <people (deletePerson)="delete($event)"></people>
```

Comment:
Do stick with the &lt, otherwise the code wont be rendered correctly

Reply:
My bad... Overlooked it while I was merging my changes... Will change it back. ;)